.github/workflows/generate_matrix_page.yml

```
@@ -0,0 +1,83 @@
+        description: 'Branch to check out'
+        default: 'main'
+        required: false
+        type: string
+      pr_number:
+        description: 'PR number to process (must be specified as a number or "all")'
+        required: true
+        type: string
+  
+jobs:
+  generate-matrix:
+    if: github.event_name != 'pull_request' || github.repository == 'rh-ecosystem-edge/nvidia-ci' 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)